learning-team/2020-10/2020-10-29.md

```diff
@@ -0,0 +1,51 @@
+# October 29, 2020
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [ ] Amy Lam (AL)
+- [ ] Chris Manson (CM)
+- [ ] Isaac Lee (IL)
+- [ ] Jared Galanis (JG)
+- [ ] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [ ] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [] Your name here (INITIALS)
+
+## Weekly review
+
+### Review action items (weekly)
+See: [Action items board](https://github.com/orgs/ember-learn/projects/47)
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/33)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+### Follow-up to Ember Language Server (ELS) discussion (IL, 15 min)
+
+On October 28, I met with Aleksandr (`@lifeart` on Discord), the main maintainer of Unstable ELS, to address possible concerns and discuss a possible path for the future. Let's catch up and discuss how we can help with logistics.
+<!-- replace with topic -->
+<!-- replace with topic -->
```

Comment:
Placeholder for @mansona / @locks (regarding upgrade.emberjs.com)

```suggestion
### Review subdomains (CM, ?? min)

<!-- Provide description -->
```

Reply:
I added this directly to the file 👍